client/js/widgets/scoreboard.js

```diff
@@ -58,15 +58,16 @@ class Scoreboard extends Widget {
       if(Array.isArray(seats))
         players = seats.map(function(s) { return { value: s.get('id'), text: s.get('player') || '-' }; });
       else { // Teams
-        for (const team in seats) 
+        for (const team in seats)
           players = players.concat(seats[team].map(function(s) { return { value: s.get('id'), text: `${s.get('player') || '-'} (${team})` } }))
       }
 
       let rounds = this.getRounds(seats, scoreProperty, 1).map(function(r, i) { return { text: r, value: i+1 }; });
+      const everyPlayerFilledLatestRound = !seats.map(s=>(s.get(scoreProperty) || []).length != rounds.length - 1).reduce((a,b)=>a||b);
```

Comment:
Could have an issue here if a seat has something other than an array in the scoreProperty.

Reply:
Also, reduce function will fail if seats is an empty array (can fix by providing an initial value to reduce).

Reply:
> Could have an issue here if a seat has something other than an array in the scoreProperty.

I don't think so. If it isn't an array, then there is just a totals line and no rounds.  And the change still works when there is just a totals line.